administration/src/cards/extensions/BirthdayExtension.tsx

```diff
@@ -13,35 +13,38 @@ const minBirthday = new PlainDate(1900, 1, 1)
 const getInitialState = (): BirthdayExtensionState => ({ birthday: null })
 
 const BirthdayForm = ({ value, setValue, isValid }: ExtensionComponentProps<BirthdayExtensionState>): ReactElement => {
-  const { viewportSmall } = useWindowDimensions()
-  const inputColor = value === getInitialState() ? Colors.GRAY1 : Colors.BLACK
-  const formStyle = viewportSmall
-    ? { fontSize: 16, padding: '9px 10px', color: inputColor }
-    : { fontSize: 14, padding: '6px 10px', color: inputColor }
+  const { birthday } = value
+  const getErrorMessage = (): string | null => {
+    if (!birthday) {
+      return 'Bitte geben Sie ein gültiges Geburtsdatum an.'
+    }
+    if (birthday.isAfter(PlainDate.fromLocalDate(new Date()))) {
+      return 'Das Geburtsdatum darf nicht in der Zukunft liegen.'
+    }
+    return null
+  }
 
-  const birthdayDate = value.birthday?.toString() ?? ''
+  const changeBirthday = (date: Date | null) => {
+    if (date) {
+      try {
+        setValue({ birthday: PlainDate.fromLocalDate(date) })
+      } catch {
+        // Couldn't parse date, no need to catch
+      }
+    }
+  }
```

Comment:
:wrench: Please use `PlainDate.safeFromCustomFormat(date)` to avoid the null check/try-catch block